jupyterhub/handlers/pages.py

```diff
@@ -591,6 +606,69 @@ def get(self, *args):
         self.finish()
 
 
+class Pagination(BaseHandler):
+
+    _page_name = 'page'
+    _per_page_name = 'per_page'
+    _default_page = 1
+    _default_per_page = 100
+    _max_per_page = 250
+    _record_name = 'users'
+    _display_msg = (
+        'Displaying {record_name} <b>{start} - {end}</b>. Total {record_name}: {total}'
+    )
+
+    def __init__(self, *args, **kwargs):
+        """Potential parameters.
+        **url**: URL in request
+        **page**: current page in use
+        **per_page**: number of records to display in the page. By default 100
+        **total**: total records considered while paginating
+        **display_msg**: informative text for pagination
```

Comment:
I'm not sure whether the display aspects (display_msg, record_name) belong in our Pagination class, since it doesn't do the rendering like the imported one did. What do you think? It might be simpler to put the text parts directly in the template, but I might be missing a benefit of having it here.

Reply:
Since we can use this pagination class from other methods (and not only for the admin template), I agree. Removing them.